app/case/routes.py

```diff
@@ -29,3 +29,10 @@ def create_case():
     def start_page():
 
         return views.Start().render()
+
+    @blueprint.route('/case/<case_id>/deed/new')
```

Comment:
The create deed transaction is going to be a full protracted transaction for conveyancers so can you move this in to a new blueprint that will contain all the "create deed" stuff.